### ISSUE TITLE
Account registration form: Customer user attributes are not automatically refilled after error

### ISSUE THREAD

**Issue by lbajsarowicz** [magento/magento2#16440]
### Preconditions
1. PHP 7.1 with Magento 2.2.3
2. Additional Customer attribute added with following setup script:
```
$this->eavSetup->addAttribute(
            CustomerMetadataInterface::ENTITY_TYPE_CUSTOMER,
            'myattribute',
            [
                'label' => 'MyAttribute',
                'type' => 'static',
                'required' => 0,
                'default_value' => '',
                'global' => ScopedAttributeInterface::SCOPE_STORE,
                'system' => 0,
                'input_filter' => 'striptags',
                'sort_order' => 5,
                'position' => 5,
                'user_defined' => 1,
                'is_used_in_grid' => 1,
                'is_filterable_in_grid' => 1,
                'is_searchable_in_grid' => 1
            ]
        );

        $this->eavSetup->addAttributeToSet(
            CustomerMetadataInterface::ENTITY_TYPE_CUSTOMER,
            CustomerMetadataInterface::ATTRIBUTE_SET_ID_CUSTOMER,
            null,
            'myattribute'
        );

        $attribute = $this->eavConfig->getAttribute(Customer::ENTITY, 'myattribute');
        $attribute->setData('used_in_forms', [
            'adminhtml_customer',
            'checkout_register',
            'customer_account_create',
            'customer_account_edit'
        ]);
        $this->attributeResource->save($attribute);
```

### Steps to reproduce
1. Enter the `customer/account/create` form and fill the form, but having an error in one of the fields

### Expected result
1. After page reload I expect error message, but also all fields filled including the custom attribute

### Actual result

1. After page reload all the fields are filled except the Custom Attribute field

**Comment by magento-engcom-team** [comment]
Hi @lbajsarowicz. Thank you for your report. 
To help us process this issue please make sure that you provided the following information:
- [ ] Summary of the issue
- [ ] Information on your environment
- [ ] Steps to reproduce
- [ ] Expected and actual results

Please make sure that the issue is reproducible on the vanilla Magento instance following **Steps to reproduce**. To deploy vanilla Magento instance on our environment, please, add a comment to the issue: 

`@magento-engcom-team give me {$VERSION} instance` 

where `{$VERSION}` is [version tags](https://github.com/magento/magento2/tags) (starting from 2.2.0+) or develop branches (2.2-develop +). 
For more details, please, review the [Magento Contributor Assistant](https://github.com/magento/magento2/wiki/Magento-Contributor-Assistant) documentation.

@lbajsarowicz do you confirm that you was able to reproduce the issue on vanilla Magento instance following steps to reproduce?
- [ ] yes
- [ ] no

**Comment by lbajsarowicz** [comment]
@magento-engcom-team give me 2.2.3 instance

**Comment by magento-engcom-team** [comment]
Hi @lbajsarowicz. Thank you for your request. I'm working on Magento 2.2.3 instance for you

**Comment by magento-engcom-team** [comment]
Hi @lbajsarowicz, here is your [Magento instance](https://i-16440-2-2-3.engcom.dev.magento.com).
Admin access: [https://i-16440-2-2-3.engcom.dev.magento.com/admin](https://i-16440-2-2-3.engcom.dev.magento.com/admin)
Login: `admin` Password: `123123q`
Instance will be terminated in up to 3 hours.

**Comment by lbajsarowicz** [comment]
@magento-engcom-team  I cannot manage Customer attributes on provided Vanilla instance, so I'm unable to reproduce the issue on your test server.

**Comment by ghost** [comment]
Helllo @lbajsarowicz thank you for your report.
Can you edit the precondition steps set it like: 
1->Create custom module
2->Create setup file InstallData.php
3->Define the install() method
4->Create custom attribute
5->Now, let run command line to install the module: php magento setup:upgrade and php bin/magento setup:static-content:deploy 
The fix for this issue is already available in the 2.3-develop branch.
We've acknowledged the issue and added to our backlog.

**Comment by lbajsarowicz** [comment]
Thanks for the information about the fix.

**Comment by magento-engcom-team** [comment]
Hi @lbajsarowicz, @geet07. 

Thank you for your report and collaboration!

The related internal Jira ticket `MAGETWO-93607` was closed as `non-reproducible` in `2.2.x`.
It means that Magento team either unable to reproduce this issue using provided _Steps to Reproduce_ from the **_Description section_** on clean or the issue has been already fixed in the scope of other tasks.

But if you still run into this problem please update or provide additional information/steps/preconditions in the **_Description section_** and reopen this issue.

**Comment by ihor-sviziev** [comment]
@sdzhepa @gabrieldagama was this issue closed by mistake, you closed it as can't reproduce or was it fixed?
